docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -261,6 +263,11 @@ screen should appear. This tutorial was tested with the "X" quadcopter frame, un
 
     Mission Planner Frame Type Selection
 
+.. note:: 
+    The following sections will explain how to set up :ref:`PWM Parameters <setting_pwm_parameters>` and :ref:`DSHOT Parameters <setting_dshot_parameters>`. Please navigate to the appropriate section for your flight controller if you are looking for additional information.
```

Comment:
I don't quite understand the purpose of this note. What is it meant to be achieving in this position? It's already deep in the Ardupilot section, so if its meant to direct people to PX4 its too late. Is it meant to say either do the PWM or DSHOT setup depending on what you want? I'm not getting that from it currently